python/python_helper_sympy.py

```diff
@@ -1,30 +1,70 @@
 import sympy
 import ast
-import sys
+from dataclasses import dataclass
+from itertools import chain
+from typing import Any, cast, Dict, List, Tuple, TypedDict, Literal, Optional
+
+SympyMapT = Dict[str, Any]
+ASTWhitelistT = Tuple[Any, ...]
```

Comment:
All AST nodes probably inherit from some ast node type, right? Should be able to do `Tuple[ast.AST, ...]` or something? Actually, wouldn't it make more sense for it to be `List[ast.AST]` anyway?

Reply:
The tuple enforces immutability. I will try and get the base ast node to work though.